src/github.com/getlantern/lantern-ui/sass/_mixins.sass

```diff
@@ -10,19 +10,19 @@
 // bold == 700
 
 =thin
-  font-family: 'Roboto Thin'
+  font-family: 'Helvetica Light', 'Segoe UI Light', Arial
```

Comment:
Helvetica only available on Mac. Segoe UI is included since Windows 7. For Ubuntu and Windows XP, Arial will take effect.

Ref http://snook.ca/archives/html_and_css/windows-subs-helvetica-arial